arduino/examples/ESP32IOBoard/ESP32IOBoard.ino

```diff
@@ -25,69 +25,129 @@
  * POSSIBILITY OF SUCH DAMAGE.
  *
  * \file ESP32IOBoard.ino
- * 
+ *
  * Main file for the io board application on an ESP32.
- * 
+ *
  * @author Mike Dunston
  * @date 13 January 2019
  */
 
 #include <Arduino.h>
-#include <WiFi.h>
-#include <SPIFFS.h>
 #include <ESPmDNS.h>
+#include <SPIFFS.h>
+#include <WiFi.h>
 #include <vector>
 
 #include <OpenMRN.h>
 #include <openlcb/TcpDefs.hxx>
+
+#include "openlcb/MultiConfiguredConsumer.hxx"
+
+// Pick one mode below, if you select USE_WIFI it will expose this node on WIFI
+// if you select USE_CAN, this node will be available on CAN.
```

Comment:
Can I pick both? Consider documenting.

Reply:
Yes, I'll update the comment as it does work for both.

Reply:
updated.